source/manual/upgrade-rails.html.md

```diff
@@ -29,9 +29,16 @@ will ensure the classes are loaded in both production and development. See
 
 ### Airbrake
 
-Upgrading to Rails 5 requires upgrading the Airbrake gem. This breaks some compatibility with
+Upgrading to Rails 5 requires upgrading the Airbrake gem to avoid deprecation 
+warnings about middleware. However, Airbrake 5 breaks some compatibility with
 our Errbit installation, namely that deployments are no longer recorded.
 
+Instead of upgrading to a Airbrake 5 release gem you can use [a branch in our
```

Comment:
`upgrading to a released version of the Airbrake 5 gem` maybe?  (There's more than one version so *the* doesn't seem right, but what I had before is equally clumsy).